src/Route/RouteGeneratorInterface.php

```diff
@@ -35,7 +35,9 @@ public function generateUrl(AdminInterface $admin, $name, array $parameters = []
      * @param array<string, mixed> $parameters
      * @param int                  $referenceType
      *
-     * @return array
+     * @return array<string, mixed>
+     *
+     * @phpstan-return array{route: string, routeParameters: array, routeAbsolute: bool}
```

Comment:
Can't we add precision for routeParameters ?

Reply:
No, because these are unspecific route parameter from the symfony router

Reply:
I was expecting something like array<string, mixed> ; but maybe I'm wrong.